modules/customize-certificates-understanding-service-serving.adoc

```diff
@@ -15,5 +15,6 @@ algorithm to generate service certificates.
 The generated certificate and key are in PEM format, stored in `tls.crt`
 and `tls.key` respectively, within a created secret. The
 certificate and key are automatically replaced when they get close to
-expiration. The service CA certificate, which signs the service
-certificates, is only valid for one year after {product-title} is installed.
+expiration.
+
+The service CA certificate, which signs the service certificates, is valid for 14 months and is automatically rotated when there is less than six months validity left. After rotation, the previous service CA configuration is still trusted until its expiration. This allows a grace period for all affected services to refresh their key material before the expiration. If you do not upgrade your cluster during this grace period, you might need to manually restart services to avoid failures after the previous service CA expires.
```

Comment:
signs -> issues

Reply:
maybe mention that upgrading restarts services and thus refreshes their key material?

Also, not sure what editor you're using but it would be easier to review if this block was broken up into separate lines.

Reply:
Thanks, will update.

And we removed hard wrapping from our guidelines a bit ago. The minor inconvenience during reviews was outweighed by by other benefits to not hard wrapping our doc text.